src/App.tsx

```diff
@@ -1,83 +1,62 @@
-import CardPreview from './components/CardPreview/CardPreview';
-import CardInputSection from './components/cardInfoForm/CardInputSection/CardInputSection';
-import CardNumberField from './components/cardInfoForm/CardNumberField/CardNumberField';
-import CardValidityPeriodField from './components/cardInfoForm/CardValidityPeriodField/CardValidityPeriodField';
-import CardCVCField from './components/cardInfoForm/CardCVCField/CardCVCField';
+import { useState } from 'react';
 import styled from '@emotion/styled';
-import useCardNumber from './hooks/useCardNumber';
-import useCardValidityPeriod from './hooks/useCardValidityPeriod';
-import useCardCVC from './hooks/useCardCVC';
+import CardPreview from './components/CardPreview/CardPreview';
+import CardForm from './components/cardInfoForm/CardForm/CardForm';
+import { CARD_COMPANY_NAME } from './components/constants/cardCompany';
 
-function App() {
-  const { cardNumber, onChange, isError, errorMessage } = useCardNumber();
+export const CARD_IFNO_INPUT_STEP = {
+  cardCompany: 'cardCompany',
+  cvc: 'cvc',
+  password: 'password',
+} as const;
```

Comment:
key, value가 동일하고, 앞으로도 동일할 예정으로 보이네요!
그래서 아래처럼 타입으로 관리해도 괜찮을 것 같아요~!

```suggestion
export type CARD_IFNO_INPUT_STEP = 'cardCompany' | 'cvc' | 'password';
```